sdk_container/src/third_party/coreos-overlay/coreos/config/env/sys-apps/zram-generator

```diff
@@ -0,0 +1,8 @@
+cros_pre_src_unpack_coreos_unpack() {
```

Comment:
Ditto, probably should be a part of a different PR.

Reply:
I merged the zram-generator PR before this one, so it now fits this one because we implement the "hack" here.